src/configure/helper/telemetryHelper.ts

```diff
@@ -0,0 +1,61 @@
+import { IActionContext, ITelemetryReporter } from "vscode-azureextensionui";
+
+import { extensionVariables } from "../model/models";
+import { TelemetryKeys } from '../resources/telemetryKeys';
+
+const uuid = require('uuid/v4');
+
+export class TelemetryHelper {
+    private actionContext: IActionContext;
+    private telemetryReporter: ITelemetryReporter;
+    private journeyId: string;
+    private command: string;
+    constructor(actionContext: IActionContext, command: string) {
+        this.actionContext = actionContext;
+        this.telemetryReporter = extensionVariables.reporter;
+        this.journeyId = uuid();
+        this.command = command;
+        this.setTelemetry(TelemetryKeys.Command, command);
```

Comment:
is there any property which can point us to an unique Azure user?

Reply:
We do not currently have any such property here as we do not identify the user.

Reply:
Can we not get some PUID or OID? It will help us filter the telemetry/traces for a particular user who has reported a problem to us.

Reply:
We do not have these ID from the credentials object of Azure. Logging `UserId` for now, the value is `username@email.com`

Reply:
As per discussion i'll roll this back and send journeyId for e2e id.